package/bleparser/almendo.py

```diff
@@ -0,0 +1,76 @@
+"""Parser for Almendo bluSensor BLE advertisements"""
+# black -l 79
+import logging
+from struct import unpack
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def parse_almendo(self, data, source_mac, rssi):
+    """Almendo parser"""
+    result = {
+        "mac": "".join(f"{x:02X}" for x in source_mac[:]),
```

Comment:
```suggestion
        "mac": to_unformatted_mac(source_mac),
```

(see also last comment, this `to_unformatted_mac` function is defined in helpers.py